pkgs/os-specific/linux/kernel/linux-xanmod.nix

```
@@ -15,6 +15,13 @@ buildLinux (args // rec {
     sha256 = "sha256-eFIWlguU1hnkAgTbRxSMTStq0X7XW4IT1/9XlQSgdMQ=";
   };
 
+  structuredExtraConfig = with lib.kernel; {
+    PREEMPT = lib.mkForce yes;
```

Additional changes, so all the features mentioned on the homepage are applied (WIP untested):

```suggestion
  
    # Preemptive Full Tickless Kernel at 500Hz.
    PREEMPT = lib.mkForce yes;
    PREEMPT_VOLUNTARY = lib.mkForce no;
    NO_HZ_FULL = yes;
    HZ_500 = yes;

    # Google's Multigenerational LRU Framework.
    LRU_GEN = yes;

    # Google's BBRv2 TCP congestion Control.
    DEFAULT_BBR2 = yes;
    DEFAULT_TCP_CONG = freeform "bbr2";
    TCP_CONG_BBR2 = yes;

    # FQ-PIE Packet Scheduling.
    NET_SCH_DEFAULT = yes;
    DEFAULT_FQ_PIE = yes;
    DEFAULT_NET_SCH = freeform "fq_pie";

    # Graysky's additional CPU optimizations for GCC / Clang.
    CC_OPTIMIZE_FOR_PERFORMANCE_O3 = yes;

    # Android Ashmem and Binder IPC Driver as module for Anbox.
    ANDROID = yes;
    ANDROID_BINDERFS = module;
    ANDROID_BINDER_DEVICES = freeform "binder,hwbinder,vndbinder";
    ANDROID_BINDER_IPC = module;
    ASHMEM = module;
```

---

I submitted the above review by accident, this PR can be merged as is.

I'll open a seperate PR once I have built and tested the kernel with these changes.

---

Would appreciate it, thanks!